src/protobot_2dnav/CMakeLists.txt

```diff
@@ -0,0 +1,204 @@
+cmake_minimum_required(VERSION 3.0.2)
+project(protobot_2dnav)
```

Comment:
is it possible to change this package name? 2dnav is a bit all-encompassing since 2dnav would usually include things like perception. Since this is move_base I'd prefer something relating to cognition (i.e protobot_planning/planner)

Reply:
sure